e2e/cypress/integration/lms_enrollment_spec.js

```diff
@@ -0,0 +1,98 @@
+// LMS enrollment tests
```

Comment:
Could we set the retries number as a global value? WDYT?

Reply:
Great Idea) Thanks) Updated this part)